wled00/const.h

```diff
@@ -157,6 +157,7 @@
 #define AP_BEHAVIOR_NO_CONN               1     //Open when no connection (either after boot or if connection is lost)
 #define AP_BEHAVIOR_ALWAYS                2     //Always open
 #define AP_BEHAVIOR_BUTTON_ONLY           3     //Only when button pressed for 6 sec
+#define AP_BEHAVIOR_BOOT_NO_CONN_5MIN     4     //Open AP when no connection after boot but only for 5min
```

Comment:
I don't think the parameters of the mode should be baked into the mode's identifier. I suggest `AP_BEHAVIOR_BOOT_TEMPORARY`.